src/nunavut/lang/properties.ini

```diff
@@ -6,6 +6,7 @@ has_standard_namespace_files = False
 namespace_is_composite_type = False
 support_namespace = nunavut.support
 indent = 4
+prefer_system_includes = True
```

Comment:
Not 100% sure if this is valid. I think the reasoning in the commit message is correct though.